rekcurd/utils/__init__.py

```diff
@@ -72,21 +72,24 @@ def __init__(self, label: PredictLabel, score: PredictScore, option: dict = None
 class EvaluateResult:
     def __init__(self, num: int = None, accuracy: float = None,
                  precision: List[float] = None, recall: List[float] = None,
-                 fvalue: List[float] = None, option: Dict[str, float] = {}):
+                 fvalue: List[float] = None, option: Dict[str, float] = {},
+                 label: List[PredictLabel] = None):
         if num is None:
             self.num = 0
             self.accuracy = 0.0
             self.precision = [0.0]
             self.recall = [0.0]
             self.fvalue = [0.0]
             self.option = {}
+            self.label = [0.0]
```

Comment:
Is it OK to set `list[float]` here?

Reply:
Yes.`PredictLabel` accepts `List[float]` and IO (grpc message)'s tensor's val is `repeated float` .

Reply:
I mean that the return type of `label` depends on the user's predictor as I wrote on `rekcurd_worker_servicer.py`.

Reply:
@keigohtr 
I removed default None value from EvaluateResult!